src/device-registry/utils/create-site.js

```diff
@@ -817,29 +817,24 @@ const manageSite = {
         );
       }
 
-      // const responseFromNearestWeatherStation = await manageSite.findNearestWeatherStation(
```

Comment:
re-enabled the fetc weather stations data --- after the external system was fixed.